### PR TITLE
test: add chaos test on network delay between apisix and etcd

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Signed-off-by: yiyiyimu <wosoyoung@gmail.com>

### What this PR does / why we need it:
add chaos test on network delay between apisix and etcd

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: t/chaos/killetcd/killetcd.go [hunk 1]
```diff
@@ -90,15 +84,30 @@ var _ = ginkgo.Describe("Test Get Success When Etcd Got Killed", func() {
 	defer ginkgo.It("restore test environment", func() {
 		stopChan <- true
 		cliSet.CtrlCli.Delete(context.Background(), getEtcdKillChaos())
+
+		resp := utils.SetRouteIgnoreError(e)
```
[the remark this answers is not in the document]
Comment (Yiyiyimu) [reply t/chaos/killetcd/killetcd.go]: Thanks! fixed

File: t/chaos/delayetcd/delayetcd.go [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+		timeStart := time.Now()
+		chaos := getEtcdDelayChaos(30)
+		err := cliSet.CtrlCli.Create(ctx, chaos)
+		gomega.Expect(err).To(gomega.BeNil())
+		time.Sleep(1 * time.Second)
+
+		defer func() {
+			err = cliSet.CtrlCli.Delete(ctx, chaos)
+			gomega.Expect(err).To(gomega.BeNil())
+			time.Sleep(1 * time.Second)
```
Comment (tokers) [inline t/chaos/delayetcd/delayetcd.go:124]: The sleep mechanism might not be so reliable, I'm afraid it's unstable in the Github Action.

We may try to make sure the environment is clean before each case runs.

Comment (Yiyiyimu) [reply t/chaos/delayetcd/delayetcd.go]: Thanks fixed. PTAL

File: t/chaos/utils/utils.go [hunk 3]
```diff
@@ -219,6 +223,25 @@ func GetSilentHttpexpectClient() *httpexpect.Expect {
 	})
 }
 
+func CheckMethodSucceed(e *httpexpect.Expect, method string, interval int) {
```
[the remark this answers is not in the document]
Comment (Yiyiyimu) [reply t/chaos/utils/utils.go]: Thanks fixed

**Review by tokers — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**